x/profiles/keeper/keeper_chain_link.go

```diff
@@ -0,0 +1,121 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/desmos-labs/desmos/x/profiles/types"
+)
+
+// StoreChainLink stores the given chain link inside the current context.
+// It assumes that the given chain link has already been validated.
+func (k Keeper) StoreChainLink(ctx sdk.Context, user string, link types.ChainLink) error {
+
+	srcAddrData, err := types.UnpackAddressData(k.cdc, link.Address)
+	if err != nil {
+		return err
+	}
+
+	target := srcAddrData.GetAddress()
+	if _, found := k.GetAccountByChainLink(ctx, link.ChainConfig.Name, target); found {
+		return fmt.Errorf("chain link already exists")
+	}
+
+	// check target address has a profile or not
+	if link.ChainConfig.Name == "desmos" {
+		if _, err := sdk.AccAddressFromBech32(target); err != nil {
+			return err
+		}
+		_, found, err := k.GetProfile(ctx, target)
+		if err != nil {
+			return err
+		}
+		if found {
+			return fmt.Errorf("the target address has profiles already")
+		}
+	}
+
+	// Check if address has the profile
+	profile, found, err := k.GetProfile(ctx, user)
+	if err != nil {
+		return err
+	}
+	if !found {
+		return fmt.Errorf("address does not have any profile")
+	}
+	// Store chain link to the profile
+	profile.ChainsLinks = append(profile.ChainsLinks, link)
+	if err := k.StoreProfile(ctx, profile); err != nil {
+		return err
+	}
+
+	store := ctx.KVStore(k.storeKey)
+	key := types.ChainsLinksStoreKey(link.ChainConfig.Name, target)
+	store.Set(key, profile.GetAddress())
+	return nil
+}
+
+// GetAccountByChainLink returns the account corresponding to the given address and the given chain name inside the current context.
+func (k Keeper) GetAccountByChainLink(ctx sdk.Context, chainName string, address string) (account sdk.AccAddress, found bool) {
+	store := ctx.KVStore((k.storeKey))
+
+	bz := store.Get(types.ChainsLinksStoreKey(chainName, address))
+	if bz == nil {
+		return nil, false
+	}
+	return account, true
```

Comment:
We are returning the `account` variable here, which is never initialized. This should return `sdk.AccAddress(bz)` instead. I suggest you removing the named returned values and simply return `(sdk.AccAddress, bool)` instead to avoid such errors